src/components/DragDrop/DragDropTypes.ts

```diff
@@ -17,9 +17,10 @@ export interface Containers {
 }
 
 export interface ContainerType {
+  className?: string;
+  columnClassName?: string;
+  emptyContent?: React.ReactNode;
+  header?: React.ReactNode;
```

Comment:
Having types in this separate file and no comments is one result of not using `children`, an example of a need for refactor